embedded-redis/src/main/java/com/playtika/test/redis/EnvUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.playtika.test.redis;
+
+import lombok.experimental.UtilityClass;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.testcontainers.containers.GenericContainer;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@UtilityClass
+class EnvUtils {
+    static Map<String, Object> registerRedisEnvironment(GenericContainer redis,
+                                                        int port,
+                                                        ConfigurableEnvironment environment,
```

Comment:
it's better to have ```ConfigurableEnvironment environment,``` as first argument, and then data which should be added to env

Reply:
done